daprdocs/content/en/getting-started/quickstarts/configuration-quickstart.md

```diff
@@ -0,0 +1,639 @@
+---
+type: docs
+title: "Quickstart: Configuration"
+linkTitle: Configuration
+weight: 76
+description: Get started with Dapr's Configuration building block
+---
+
+Let's take a look at Dapr's [Configuration building block]({{< ref configuration-api-overview.md >}}). A configuration item is often dynamic in nature and tightly coupled to the needs of the application that consumes it. Configuration items are key/value pairs containing configuration data, such as:
+- App ids
+- Partition keys
+- Database names, etc
+
+In this quickstart, you'll run an `order-processor` microservice that utilizes the Configuration API. The service:
+1. Gets configuration items from the configuration store.
+1. Subscribes for configuration updates.
+
+<img src="/images/configuration-quickstart/configuration-quickstart-flow.png" width=1000 alt="Diagram that demonstrates the flow of the configuration API quickstart with key/value pairs used.">
+
+Select your preferred language-specific Dapr SDK before proceeding with the Quickstart.
+
+{{< tabs "Python" "JavaScript" ".NET" "Java" "Go" >}}
```

Comment:
We are inconsistent in the docs with .NET, Dotnet and C#. What do you suggest? Should we change to .NET everywhere?

Reply:
I typically use .NET